packages/ui/src/i18n.ts

```diff
@@ -11,6 +11,7 @@ const dictEn: { [key in keyof Dict]: string } = {
   opacity: 'Opacity',
   rotate: 'Rotate',
   required: 'Required',
+  readOnly: 'Not editable',
```

Comment:
I'm still trying to think of the best description. I will keep thinking...

Reply:
I'm starting to think that just `Editable` is the best UX, ticked by default for those fields that support it (which would mean either inverting the saved setting value or changing the schema structure). 

I will get some feedback from colleagues later today.

Reply:
Is "Read Only" really that problematic? I understand that my opinion doesn't carry much weight here since I can only judge from a Japanese language perspective, but at least in Japanese, it doesn't feel awkward at all.

I apologize for making a statement that brings the discussion back to square one.

Reply:
No problem! :) 

After speaking with a couple of people, we think that this is the best (English) UX:

https://github.com/user-attachments/assets/99542e0c-cad0-491d-883c-8c95d1eed5b1

A field is 'Editable' by default. If Editable, it can be Required. If not, the Required option is not available (could be hidden or disabled) 

This feels like the most logical naming and UX design for a Designer using the form. "Read Only" feels like more of a developer term.

However, I do agree "Read Only" is better than "Not Editable", if you decide to keep this approach.

Reply:
Did it!

![CleanShot 2024-08-09 at 11 51 45](https://github.com/user-attachments/assets/7d8398db-9132-4b9b-8155-8f21f046df02)


https://github.com/pdfme/pdfme/pull/547/commits/631823b496e18d1e4282fecdb18ba11710139a1d

Reply:
Cool! 😄 

But I don't think we should see these options for non-editable components such as shapes:

![Screenshot 2024-08-09 at 05 53 29](https://github.com/user-attachments/assets/68c4e272-5904-48a9-982f-3e03d61e4fb4)

Reply:
Ops. Fixed